pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/JdbiBlockNesting.sql.stg

```diff
@@ -1,7 +1,9 @@
 group JdbiBlockNesting;
 
 queryOrderedNestedFormBlockDtosByBlockIdAndInstanceGuid() ::= <<
-select bt.block_type_code,
+select null as form_section_id,
```

Comment:
You sure about return null for sectionId?

Reply:
Yup, a nested block doesn't explicitly store the section. It does have have parent block id though.